src/core_plugins/kibana/public/management/sections/indices/edit_index_pattern/scripted_fields_table/scripted_fields_table.html

```diff
@@ -1,36 +1,59 @@
-<h3>Scripted fields</h3>
-<p>These scripted fields are computed on the fly from your data. They can be used in visualizations and displayed in your documents, however they can not be searched. You can manage them here and add new ones as you see fit, but be careful, scripts can be tricky! <!-- If you need some examples, why not let Kibana <a ng-click="addDateScripts()"><strong>create a few examples from your date fields.</strong></a --></p>
+<h3 class="kuiTextTitle kuiVerticalRhythm">
+  Scripted fields
+</h3>
 
-<header>
-  <a
-    data-test-subj="addScriptedFieldLink"
-    ng-href="{{ kbnUrl.getRouteHref(indexPattern, 'addField') }}"
-    class="kuiButton kuiButton--primary kuiButton--iconText"
-  >
-    <span aria-hidden="true" class="kuiButton__icon kuiIcon fa-plus"></span>
-    <span>Add Scripted Field</span>
-  </a>
-</header>
+<p class="kuiText kuiVerticalRhythm">
+  These scripted fields are computed on the fly from your data. They can be used in visualizations and displayed in your documents, however they can not be searched. You can manage them here and add new ones as you see fit, but be careful, scripts can be tricky!
+</p>
 
-<div class="hintbox" ng-if="getDeprecatedLanguagesInUse().length !== 0">
-  <h4>
-    <i class="fa fa-warning text-warning"></i> Deprecation Warning
-  </h4>
-  <p>
-    We've detected that the following deprecated languages are in use: {{ getDeprecatedLanguagesInUse().join(', ') }}.
-    Support for these languages will be removed in the next major version of Kibana and Elasticsearch.
-    We recommend converting your scripted fields to
-    <a target="_window" ng-href="{{docLinks.painless}}">Painless <i class="fa-link fa"></i></a>.
-  </p>
+<div
+  class="kuiInfoPanel kuiInfoPanel--warning kuiVerticalRhythm"
+  ng-if="getDeprecatedLanguagesInUse().length !== 0"
+>
+  <div class="kuiInfoPanelHeader">
+    <span class="kuiInfoPanelHeader__icon kuiIcon kuiIcon--warning fa-bolt"></span>
+    <span class="kuiInfoPanelHeader__title">
+      Deprecation Warning
+    </span>
+  </div>
+
+  <div class="kuiInfoPanelBody">
+    <div class="kuiInfoPanelBody__message">
+      We&rsquo;ve detected that the following deprecated languages are in use: {{ getDeprecatedLanguagesInUse().join(', ') }}.
+      Support for these languages will be removed in the next major version of Kibana and Elasticsearch.
+      We recommend converting your scripted fields to
+      <a class="kuiLink" target="_window" ng-href="{{docLinks.painless}}">Painless</a>.
```

Comment:
This ng-href doesn't seem to be working anymore for some reason

![screen shot 2017-04-27 at 3 56 25 pm](https://cloud.githubusercontent.com/assets/6239176/25501859/3e038202-2b62-11e7-86ff-3d64e3d86fd0.png)